### DISCUSSION TITLE
404 error for static files

### DISCUSSION THREAD

**Discussion by artemu78** [community/community#74068] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My [static github pages website](https://artemu78.github.io/rsc) respond with index.html properly, but refuse to provide some other static resources inside the folders (JS and CSS files).

For instance. JS in https://artemu78.github.io/rsc/_next/static/css/876d048b5dab7c28.css but GitHub respond with 404 for this url. But it present in GitHub repo https://github.com/artemu78/artemu78.github.io/blob/main/rsc/_next/static/css/876d048b5dab7c28.css

Same time my [another static github pages website](https://artemu78.github.io/tech_assessment/) provides nested static resources pretty well, everything works fine for other folders.

What might be a problem?

**Comment by BhathikaGimhan** [comment]
1. **Case Sensitivity**: Double-check that the file paths and file names are using the correct case sensitivity. For example, make sure that the file names in the URL match the actual file names in your repository.

2. **Check for Typos**: Ensure that there are no typos in the file paths or URLs.

3. **Folder Structure**: Verify that the files are located in the correct folders in your repository.

4. **File Permissions**: Make sure that the files have the correct permissions and are not restricted in any way.

5. **Deployment Process**: Confirm that your deployment process (such as GitHub Actions or any other CI/CD pipeline) is correctly handling the files and not excluding them.

6. **Cache Issues**: Clear your browser cache or try accessing the page in a different browser to rule out any local caching issues.

7. **GitHub Pages Configuration**: Check your repository settings to ensure that GitHub Pages is configured correctly and that the branch being used for deployment contains the necessary files.

8. **File Size Limit**: Be aware that GitHub Pages has a file size limit of 100 MB per file. If your files exceed this limit, they will not be served.

9. **Repository Permissions**: Make sure that you have the necessary permissions to access the files in your repository.

**Comment by artemu78** [comment, marked as answer]
Thank you @BhathikaGimhan issue was Jekyll.
It silently ignores all folders whose names begin with an underscore, like my `_next`.
So solution is to put ".nojekyll" file into ROOT. Now everything works.
